### PR TITLE
Added limits to MAT concept

### PR DESCRIPTION
**Pull request by armsteadj1**
## Description

- Adding limits to the MAT concept page

## Testing required outside of automated testing?

- [ ] Not Applicable

### Screenshots (if appropriate):

- [ ] Not Applicable

## Rollback / Rollforward Procedure

- [ ] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: collections/concepts/what-are-mats.md [hunk 1]
```diff
@@ -89,6 +89,10 @@ Once a Tenant exceeds the 1,000 free MATs, you will be prompted to enter a payme
 In order to facilitate your evaluation of Basis Theory, you will not be immediately blocked from continued usage of Basis Theory's services,
 and will have a short grace period to enter your payment details while continuing evaluation.
 
+## Are there usage limits to the monthly active tokens?
+
+The monthly active tokens are designed to make billing simple for almost every way that customers are using the platform. There are customer use cases where this model drives a high cost for Basis Theory to support the use case so we have alternate plans available when there are more than 10 interactions per MAT in a month.
```
Comment (djejaquino) [inline collections/concepts/what-are-mats.md:94]: Should we add a contact link for people to reach out if they have specific needs?

**Review by djejaquino — APPROVED** [review]

File: collections/concepts/what-are-mats.md (hunk 1 above)
Comment (armsteadj1) [inline collections/concepts/what-are-mats.md:94]:
```suggestion
There is a limit of 10 interactions per MAT per month. Our billing is designed to be simple for almost every use-case for using our platform platform. In some situations more than 10 interactions per MAT in a month are required. [Contact us](https://basistheory.com/contact) for alternate plans available when this limit isn’t enough.
```

**Review by djejaquino — DISMISSED** [review]

File: collections/concepts/what-are-mats.md [hunk 2]
```diff
@@ -89,6 +89,10 @@ Once a Tenant exceeds the 1,000 free MATs, you will be prompted to enter a payme
 In order to facilitate your evaluation of Basis Theory, you will not be immediately blocked from continued usage of Basis Theory's services,
 and will have a short grace period to enter your payment details while continuing evaluation.
 
+## Are there usage limits to the monthly active tokens?
+
+There is a limit of 10 interactions per MAT per month. Our billing is designed to be simple for almost every use-case for using our platform platform. In some situations more than 10 interactions per MAT in a month are required. [Contact us](https://basistheory.com/contact) for alternate plans available when this limit isn’t enough.
```
Comment (dhudec) [inline collections/concepts/what-are-mats.md:94]:
```suggestion
There is a limit of 10 interactions per MAT per month. Our billing is designed to be simple for almost every use-case for using our platform. In some situations more than 10 interactions per MAT in a month are required. [Contact us](https://basistheory.com/contact) for alternate plans available when this limit isn’t enough.
```

**Review by djejaquino — APPROVED** [review]

**Pull request merged**